packages/polytechcolloscopeclient/lib/src/polytechcolloscopeclient_base.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:beautiful_soup_dart/beautiful_soup.dart';
+import 'package:html/dom.dart';
+import 'package:requests_plus/requests_plus.dart';
+
+import 'colloscope_datastructs.dart';
+import 'consts.dart';
+
+class PolytechColloscopeClient {
+  final String username, password;
```

Comment:
Making it private would be safer.

Reply:
True but i didn't know dart syntax so i kept it simple to begin